src/form/form-label.tsx

```diff
@@ -24,6 +25,11 @@ export const FormLabel = ({
         <Label {...otherProps}>
             {children}
             {addon && addon.type && renderAddon()}
+            {typeof subtitle === "string" ? (
+                <SubLabel {...otherProps}>{subtitle}</SubLabel>
```

Comment:
Don't think you should just pass the `otherProps` in. They are meant for the `Label`. What you can do is to set a test id on it using the test id that was passed to the `Label` component